src/modules/keyboardmanager/ui/EditKeyboardWindow.cpp

```diff
@@ -63,14 +64,148 @@ void createEditKeyboardWindow(HINSTANCE hInst, KeyboardManagerState& keyboardMan
     // Update the xaml island window size becuase initially is 0,0
     SetWindowPos(hWndXamlIslandEditKeyboardWindow, 0, 0, 0, 400, 400, SWP_SHOWWINDOW);
 
-    //Creating the Xaml content
+    // Creating the Xaml content. xamlContainer is the parent UI element
     Windows::UI::Xaml::Controls::StackPanel xamlContainer;
-    xamlContainer.Background(Windows::UI::Xaml::Media::SolidColorBrush{ Windows::UI::Colors::LightGray() });
-    Windows::UI::Xaml::Controls::Button bt;
-    bt.Content(winrt::box_value(winrt::to_hstring("Don't Type key")));
-    xamlContainer.Children().Append(bt);
+    xamlContainer.Background(Windows::UI::Xaml::Media::SolidColorBrush{ Windows::UI::Colors::AliceBlue() });
```

Comment:
Can we set the background colors across the UI to some plain "accessible" color for now like either white or gray? We will anyways have to change it to match the spec later, but just to keep it consistent it's better if we stick to something basic for now.

Reply:
Sure switched the background color to `Light Gray`